File: SecondYear/Anantj2499_AnantJaiswal_2226cseml1088_2/Week_2/CheckPrime.java
public class CheckPrime {
        public static String isPrime(int num) {
            //Your code goes here
            int flag =0;
            for (int i =2; i<=Math.sqrt(num);i++){
                if(num%i==0){
                    flag=1;
                    break;
                }
            }
            if(flag==1||num==1){
                return "NO";
            }
            else
            {
                return "YES";
            }
        }
}

File: SecondYear/adityshrma08_AdityaSharma_2226csit1014_2/Week_2/Check Prime.cpp
bool isPrime(int n)
{
	// Write your code here.
	int count=0;
	if(n<=1){
		return false;
	}
	for(int i=2;i*i<n;i++){
		if(n%i==0){
			count=1;
			break;
		}
		else{
			count=0;
		}
	}
		if(count==1){
			return false;
		}
		else
		return true;
	
}
